packages/elements/src/clock/index.ts

```diff
@@ -482,28 +526,55 @@ export class Clock extends ResponsiveElement {
   * @param event Event Object
   * @returns {void}
   */
-  private handleUpKey (event: KeyboardEvent): void {
-    this.shift(UP, this.getShiftAmountFromTarget(event.target));
+  private handleUpKey (): void {
+    const selection = this.renderRoot.querySelector(`[part~=${this.activeSegment}]`);
+    this.shift(Direction.UP, this.getShiftAmountFromTarget(selection));
   }
 
   /**
   * Handle DOWN key press
   * @param event Event Object
   * @returns {void}
   */
-  private handleDownKey (event: KeyboardEvent): void {
-    this.shift(DOWN, this.getShiftAmountFromTarget(event.target));
+  private handleDownKey (): void {
+    const selection = this.renderRoot.querySelector(`[part~=${this.activeSegment}]`);
+    this.shift(Direction.DOWN, this.getShiftAmountFromTarget(selection));
+  }
+
+  /**
+  * Handles Left key press
+  * @param event Event Object
+  * @returns {void}
+  */
+  private handleLeftRightKey (): void {
+    this.activeSegment = this.activeSegment === Segment.HOURS ? Segment.MINUTES : Segment.HOURS;
+  }
+
+  /**
+  * Updates aria-value-text to have the same value as screen
+  * @returns {void}
+  */
+  private updateAriaValue () {
+    const value = `Time: ${padNumber(this.displayHours, 2)}:${padNumber(this.displayMinutes, 2)}${this.showSeconds ? ':' + padNumber(this.displaySeconds, 2) : ''}${this.amPm ? ' ' + this.displayAmPm : ''}`;
```

Comment:
Can this be broken down into smaller statements ?

Reply:
Updated by break the line.